interfaces/PA-App/src/assets/i18n/en.json

```diff
@@ -71,9 +71,9 @@
     },
     "registration-summary": {
       "documents": "Make sure to have these documents ready:",
-      "no-validation-conclusion": "Thank you. Your registration is now complete.   <br><br>   You should receive an SMS on the provided phone number shortly that confirms your registration.   <br><br>   You will be informed if your application has been successful within 14 days.   <br><br>   This is the final message for now.",
+      "no-validation-conclusion": "Thank you. Your registration is now complete.   <br><br>   You should receive an SMS/Whatsapp on the provided phone number shortly that confirms your registration.   <br><br>   You will be informed if your application has been successful within 14 days.   <br><br>   This is the final message for now.",
```

Comment:
@RubenGeo I think Elwin always says the correct spelling is "whatsApp".

Reply:
It's not me. Its Meta/Facebook/WhatsApp itself that says the brand-name is "WhatsApp"
See: https://www.whatsapp.com/about

Reply:
Not WhatsApp? 
![image](https://user-images.githubusercontent.com/34537157/146020426-10dca0a0-5351-4dcc-b31f-09b0633c3264.png)

Reply:
I guess you're right